src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConfiguration.java

```diff
@@ -78,6 +84,21 @@ ConsumerConfig getRetryConsumerConfig() {
         return config;
     }
 
+    @Bean("error-consumer-group")
+    @Lazy
+    CHKafkaConsumerGroup getErrorConsumer() {
+        return new CHKafkaConsumerGroup(getErrorConsumerConfig());
+    }
+
+    ConsumerConfig getErrorConsumerConfig() {
+        ConsumerConfig config = new ConsumerConfig();
+        config.setBrokerAddresses(new String[]{brokerAddress});
+        config.setTopics(Collections.singletonList(errorTopicName));
+        config.setPollTimeout(pollTimeout);
+        config.setGroupName(errorConsumerGroupName);
```

Comment:
i think we should also explicitly set config.setAutoCommit(false). we'll need to add it to the other consumers at some point.

Reply:
would it be worth doing that in a seperate pr when we do all of them?

Reply:
ok